testhelpers/org.eclipse.n4js.ui.tests.helper/src/org/eclipse/n4js/tests/builder/AbstractBuilderTest.java

```diff
@@ -78,10 +77,15 @@
 	private IResourceSetProvider resourceSetProvider;
 	@Inject
 	private ResourceDescriptionsProvider resourceDescriptionsProvider;
+
+	/**
+	 * Allows to trigger operations on the workspace that are considered to prevent race conditions, e.g. allows to
+	 * build synchronously.
+	 */
 	@Inject
-	private ExternalLibraryBuildScheduler externalLibraryBuildJobProvider;
-	@Inject
-	private ProjectOpenedOrClosedListener projectOpenedOrClosedListener;
+	@Rule
+	public TestedN4JSWorkspace testedWorkspace;
```

Comment:
Could you please briefly summarize the current (i.e. after this PR is merged) state of porting tests to the new `TestedWorkspace`. How many tests have you ported over?

Reply:
Almost all affected tests that I found do inherit from AbstractBuilderTest. In theory it would be good to search for clients of all the static test utils and refactor them piece by piece. It is a lot of effort though that is probably worth to be tackled incrementally. As a rule of thumb: Use the testedWorkspace instead of the static utils and enhance the tested workspace with functionality where needed.